gym-libs compile fails on debian

### Expected behavior
make install to complete compile from cloned .git

### Current behavior
Cmake .. completes without errors, but make install fails, with "No such file or directory" message:

root@OpenVAS:~/gvm-libs/build# cmake ..
-- Configuring the Greenbone Vulnerability Management Libraries...
-- Looking for clang-format...
-- Looking for clang-format... /usr/bin/clang-format
-- Install prefix: /usr/local
-- Checking for module 'hiredis>=0.10.1'
--   
-- Looking for gpgme...
-- Found gpgme: /usr/lib/x86_64-linux-gnu/libgpgme.so, version 1.8.0
-- Looking for libgcrypt...
-- Looking for libgcrypt... /usr/lib/x86_64-linux-gnu/libgcrypt.so
-- Looking for libldap...
--   Found /usr/lib/x86_64-linux-gnu/libldap.so - ldap support enabled
-- Configuring done
-- Generating done
-- Build files have been written to: /root/gvm-libs/build


root@OpenVAS:~/gvm-libs/build# make install
[ 44%] Built target gvm_base_shared
[ 47%] Building C object util/CMakeFiles/gvm_util_shared.dir/kb.c.o
/root/gvm-libs/util/kb.c:31:81: fatal error: hiredis/hiredis.h: No such file or directory
 #include <hiredis/hiredis.h> /* for redisReply, freeReplyObject, redisCommand */
                                                                                 ^
compilation terminated.
util/CMakeFiles/gvm_util_shared.dir/build.make:158: recipe for target 'util/CMakeFiles/gvm_util_shared.dir/kb.c.o' failed
make[2]: *** [util/CMakeFiles/gvm_util_shared.dir/kb.c.o] Error 1
CMakeFiles/Makefile2:172: recipe for target 'util/CMakeFiles/gvm_util_shared.dir/all' failed
make[1]: *** [util/CMakeFiles/gvm_util_shared.dir/all] Error 2
Makefile:160: recipe for target 'all' failed
make: *** [all] Error 2
### Steps to reproduce
1. clone .git
2. Run cmake ..
3. run make install

### GVM versions
**gsa:** (gsad --version)

**gvm:** (gvmd --version)

**openvas-scanner:** (openvassd --version)

**gvm-libs:**

**openvas-smb:**

### Environment
**Operating system:**
Debian
**Installation method / source:** (packages, source installation)
git clone.

### Logfiles

--- a/util/CMakeLists.txt
+++ b/util/CMakeLists.txt
@@ -45,7 +45,7 @@ pkg_check_modules (UUID REQUIRED uuid>=2.25.0)
 pkg_check_modules (LIBSSH REQUIRED libssh>=0.6.0)
 
 # for kb we need libhiredis
-pkg_check_modules (REDIS hiredis>=0.10.1)
+pkg_check_modules (REDIS REQUIRED hiredis>=0.10.1)
 
 # Set NVTICACHE name with the version
 set (NVTICACHE_STR "nvticache${PROJECT_VERSION}")

